calc/HamiltonianOne.cpp

```diff
@@ -407,7 +407,7 @@ void HamiltonianOne::build() {
 #pragma omp parallel for schedule(static, 1)
 
     // Loop through steps
-    for (size_t step = 0; step < nSteps; ++step) {
+    for (int step = 0; step < static_cast<int>(nSteps); ++step) {
```

Comment:
Could we then at least do
```
int nSteps_i = static_cast<int>(nSteps);
for (int step = 0; step < nSteps_i; ++step) {
```
to not have the runtime cost of a `static_cast` at every iteration or we just leave it as
```
for (int step = 0; step < nSteps; ++step) {
```
and tolerate the compiler warning.